### PR TITLE
Add the context menu after the welcome window

### PR DESCRIPTION
**Pull request by 0x6431346e**
## What changed?
Insert here the changes.

### CODE REVIEW

File: app/index.js [hunk 1]
```diff
@@ -111,13 +113,28 @@ function startWelcome () {
...
       app.quit()
     })
   })
 }
 
+/**
+ * This adds the option "Add to IPFS via Orion" to the context menu of the OS, for files 
+ * and directories. Currently only on windows.
+ */
+function enableContextMenu () {
```
Comment (koalalorenzo) [inline app/index.js]: Should we move this function into `app/lib`?

**Comment by 0x6431346e** [comment]
But then there would be absolutely no way to remove it, do we want that?

**Comment by koalalorenzo** [comment]
We will add a new task to add a way to remove it

File: app/index.js [hunk 2]
```diff
@@ -111,6 +113,7 @@ function startWelcome () {
     app.mainWindow = welcomeWindow
     welcomeWindow.on('closed', () => {
       if (Settings.get('welcomeVersion') === 1) {
+        enableContextMenu()
```
Comment (koalalorenzo) [inline app/index.js]: Can we add a small comment here? What is this doing here? :)

File: app/windows/Settings/Components/ConnectivityPanel.jsx [hunk 3]
```diff
@@ -5,7 +5,7 @@ import { trackEvent } from '../../../stats'
 
 import { Pane, Button, CheckBox } from 'react-photonkit'
 import Input from '../../../components/Input'
-import * as WinShell from '../../../util/win-shell'
+import * as WinShell from '../../../lib/os-context-menu/win-shell'
```
Comment (koalalorenzo) [inline app/windows/Settings/Components/ConnectivityPanel.jsx]: <3 <3 <3

**Pull request merged**